observations/forward_operators/obs_def_upper_atm_mod.f90

```diff
@@ -323,8 +343,8 @@ subroutine get_expected_gnd_gps_vtec(state_handle, ens_size, location, obs_val,
    end if
 end do
 
-! clear the error from the last level and start again?
-tec=0.0_r8 !start with zero for the summation
+! Set all ensemble members tec to zero for summation
+tec=0.0_r8
```

Comment:
line 365 there is a single precision literal 10.0

obs_val = tec * 10.0**(-16)
obs_val = tec * 1.0e-16_r8

1